types/db.go

```diff
@@ -67,3 +68,19 @@ type Attribute struct {
 	Key        string        `gorm:"index" json:"key"`
 	Value      string        `json:"value"`
 }
+
+// Plan is a Terraform plan
+type Plan struct {
+	ID               uint           `sql:"AUTO_INCREMENT" gorm:"primary_key" json:"id"`
+	Version          Version        `json:"version"`
+	VersionID        sql.NullInt64  `gorm:"index" json:"-"`
+	Lineage          string         `json:"lineage"`
+	TerraformVersion string         `gorm:"varchar(10)" json:"terraform_version"`
```

Comment:
This field was called `TFVersion` in the `State` struct. The `varchar(10)` is a good idea, but I'd rather keep it homogeneous with the rest of the types. We can either change that in the other structs already, or do it afterwards.